src/main/java/seedu/address/model/account/Account.java

```diff
@@ -0,0 +1,195 @@
+    }
+
+    /**
+     * Adds a profit entry to the account.
+     */
+    public void addProfit(Profit p) {
+        profits.add(p);
+    }
+
+    /**
+     * Replaces the given expense entry {@code target} in the list with {@code editedExpense}.
+     * {@code target} must exist in the account.
+     */
+    public void setExpense(Expense target, Expense editedExpense) {
+        requireNonNull(editedExpense);
+
+        expenses.setExpense(target, editedExpense);
+    }
+
+    /**
+     * Replaces the given profit entry {@code target} in the list with {@code editedProfit}.
+     * {@code target} must exist in the account.
+     */
+    public void setProfit(Profit target, Profit editedProfit) {
+        requireNonNull(editedProfit);
+
+        profits.setProfit(target, editedProfit);
+    }
+
+    /**
+     * Removes {@code key} from this {@code Account}.
+     * {@code key} must exist in the account.
+     */
+    public void removeExpense(Expense key) {
+        expenses.remove(key);
+    }
+
+    /**
+     * Removes {@code key} from this {@code Account}.
+     * {@code key} must exist in the account.
+     */
+    public void removeProfit(Profit key) {
+        profits.remove(key);
+    }
+
+    //// util methods
+
+    @Override
+    public String toString() {
+        return expenses.asUnmodifiableObservableList().size() + " expenses\n"
+                + profits.asUnmodifiableObservableList().size() + " profits";
+        // TODO: refine later
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    @Override
+    public ObservableList<Expense> getExpenseList() {
+        return expenses.asUnmodifiableObservableList();
+    }
+
+    @Override
+    public ObservableList<Profit> getProfitList() {
+        return profits.asUnmodifiableObservableList();
+    }
+
+    /**
+     * Returns true if both Account have identity fields that are the same.
+     * This defines a weaker notion of equality between two entries.
+     */
+    public boolean isSameAccount(Account otherAccount) {
+        if (otherAccount == this) {
+            return true;
+        }
+
+        return otherAccount != null
+                && otherAccount.getName().equals(getName());
+    }
+
+    /**
+     * Returns true if both expenses have the same identity and data fields.
+     * This defines a stronger notion of equality between two accounts.
+     */
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof AddressBook // instanceof handles nulls
+                && (name.equals(((Account) other).name)
+                && expenses.equals(((Account) other).expenses)
+                && profits.equals(((Account) other).profits)));
+    }
+
+    @Override
+    public int hashCode() {
+        // use this method for custom fields hashing instead of implementing your own
+        return Objects.hash(name, expenses, profits);
+    }
```

Comment:
I think should include an empty line here (the Addressook they have)

Reply:
Okay!